### ISSUE TITLE
a11y > item detail > quantity dropdown should have form field label

### ISSUE THREAD

**Issue by ryanschaefer41** [vigetlabs/EcoCart#68]
Let's add a "Quantity" form field label above the selector using the same style as is used in the nav.

![image](https://user-images.githubusercontent.com/23317410/89205585-f2da9780-d585-11ea-8ab0-9a8fa13119fa.png)
